src/components/Header.tsx

```diff
@@ -21,14 +21,16 @@ const Header = (props: BoxProps) => {
       <FluidContainer py={5}>
         <Flex justify="space-between" alignItems="center">
           <Flex alignItems="center">
-            <Box h={LOGO_SIZES} w={LOGO_SIZES} mr={2}>
-              <Image
-                priority
-                src={mascotIcon}
-                alt="Octoclairvoyant reading a crystal ball"
-                placeholder="blur"
-              />
-            </Box>
+            <RouteLink href="/">
+              <Box h={LOGO_SIZES} w={LOGO_SIZES} mr={2}>
+                <Image
+                  priority
+                  src={mascotIcon}
+                  alt="Octoclairvoyant reading a crystal ball"
```

Comment:
The alt of the image should reflect now this is a link to the home page:

`alt="Go to home page"`

Extracted from [Creating valid and accessible links](https://www.a11yproject.com/posts/2019-02-15-creating-valid-and-accessible-links/):
> To be a valid link it must have:
> ...
> Link content text describing where the link is going, this could be plain text **or the `alt` attribute of an image**.

Reply:
Good catch! I didn't know that you should use the `alt` text to describe where the link goes, when you use an image as link. 😄 Learned something new today. 😄